src/vm.rs

```diff
@@ -91,7 +92,7 @@ impl<VCpuType: VirtualCPU> UhyveVm<VCpuType> {
 		let memory_size = params.memory_size.get();
 
 		#[cfg(target_os = "linux")]
-		let mem = MmapMemory::new(0, memory_size, arch::RAM_START, params.thp, params.ksm);
+		let mem = MmapMemory::new(0, memory_size, PhysAddr::new(0x20000), params.thp, params.ksm);
```

Comment:
This looks like a separate commit

Reply:
Did my `git log -p` this time, should be good to go!